modules/admin_manual/pages/maintenance/upgrading/database_upgrade.adoc

```diff
@@ -0,0 +1,80 @@
+= Database Upgrade
+:toc: right
+:upgrade-mariadb-url: https://mariadb.com/kb/en/upgrading/
+:compressed-row-format-url: https://mariadb.com/kb/en/innodb-compressed-row-format/#read-only
+
+== Introduction
+
+Usually, when upgrading a database, follow the guides the vendor provides. You may also take a look at the xref:installation/manual_installation/manual_installation.adoc#install-a-database[Install a Database] section for general notes.
+
+This document supports an upgrade in case there are prerequisites and/or special steps to be taken to upgrade to a particular database version.
+
+== Upgrading to MariaDB 10.6
+
+ownCloud runs fine when using a MariaDB version lower than 10.6. You can upgrade ownCloud as usual without special considerations to the database.
+
+When planning to upgrade to MariaDB 10.6, some prerequisites have to be met and upgrade steps have to be taken. 
+
+This is because MariaDB 10.6 has changed that _tables that are of the COMPRESSED row format are read-only by default_. For more information see the {compressed-row-format-url}[InnoDB COMPRESSED Row Format] document.
+
+NOTE: This process only needs to be carried out once.
+
+=== Prerequisites
+
+. You must have upgraded your ownCloud installation to version 10.9 or above. To do so, follow the
+xref:maintenance/upgrading/manual_upgrade.adoc[Manual ownCloud Upgrade] guide.
+
+. Backup your ownCloud installation, especially the database. To do so, follow the
+xref:maintenance/backup_and_restore/backup.adoc[Backing up ownCloud] guide.
+
+. When running a MariaDB release lower than 10.5, you have to upgrade step-by-step for each minor release up to 10.5.x. Follow the respective {upgrade-mariadb-url}[Upgrading MariaDB] guide.
++
+[WARNING]
+====
+You must not skip minor releases of MariaDB when upgrading like from 10.3 -> 10.5, you have to upgrade to each minor version in between step by step.
+====
++
+Have a backup of you database to rollback in case of issues.
+
+. Prevent browser access +
+Depending on your setup, prevent accessing users your ownCloud instance like:
```

Comment:
```suggestion
Depending on your setup, prevent users accessing your ownCloud instance by:
```

Reply:
Maybe setting maintenance mode is enough?
When maintenance mode is on, users of the webUI should be informed if/when they try to login, and clients should "understand" that the server is in maintenance mode, and stop trying to sync files. True?

Reply:
good point, will have a deeper thought on that...

Reply:
fixed, see commit